src/openrct2-ui/drawing/engines/opengl/ApplyTransparencyShader.cpp

```diff
@@ -0,0 +1,94 @@
+#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#ifndef DISABLE_OPENGL
+
+#include "ApplyTransparencyShader.h"
+
+struct VDStruct {
```

Comment:
Brace needs to go on its own line.

Reply:
I tried to remember that for the if/else/while/for, but forgot it here. I'll fix it.